dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/dag.vue

```diff
@@ -893,4 +893,22 @@
   .operBtn {
     padding: 8px 6px;
   }
+
+  .el-drawer__body {
+    ::selection {
+      background: #409EFF;
+      color: white;
+    }
+
+    ::-moz-selection {
```

Comment:
Please remove `::-moz-selection` and `::-webkit-selection`, `postcss-loader` and `autoprefixer` will help add the prefixes when building